### ISSUE TITLE
CSS from style tag is extracted as text

### ISSUE THREAD

**Issue by woconnor** [markusmobius/go-domdistiller#1]
I've encountered a situation where go-domdistiller extracts CSS from a `<style>` element as if it's text. Ideally content is never extracted from tags like `<style>` or `<script>`.

In particular, when go-domdistiller sees this:
```
    <figure>
      <a href="http://example.com/link">
        <img src="http://example.com/image.png" />
        <style>
          figure a img {
            max-width: 100%;
          }
        </style>
      </a>
    </figure>
```

It outputs this:
```
<figure><img src="http://example.com/image.png"/><figcaption>figure a img { max-width: 100%; }</figcaption></figure>
```

I've attached a minimal file to reproduce this scenario and the corresponding output.
[input.txt](https://github.com/markusmobius/go-domdistiller/files/6207467/input.txt)
[output.txt](https://github.com/markusmobius/go-domdistiller/files/6207470/output.txt)

**Comment by RadhiFadlillah** [comment]
It should be fixed by [3e88970] (and [b82fa86] if you use stable branch). Could you check it out again?

[3e88970]: https://github.com/markusmobius/go-domdistiller/commit/3e8897050667c75b605f3e817803ba0ee445adca
[b82fa86]: https://github.com/markusmobius/go-domdistiller/commit/b82fa863d4b64b8196a6a24efda9de61e40855dc

**Comment by woconnor** [comment]
Thanks for the fix! I just verified that it works. Cheers.
